src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -18,12 +27,12 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     @track version = '';
     @api showSpinner = false;
     @api batchId;
-    @api submissions = [];
     @api hasPageLevelError = false;
     label = { messageLoading, geSave, geCancel };
     @track formTemplateId;
     erroredFields = [];
     @api pageLevelErrorMessageList = [];
+    @track _data; // Row being updated when in update mode
```

Comment:
Is there a less generic way to name this than "_data"?

Reply:
Went with _dataRow - thanks for helping me hash that out!